src/styles/index.scss

```diff
@@ -1,3 +1,131 @@
+$body-font: "Roboto", sans-serif;
+$font-size: 10px;
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-blue-accent: #00acdc;
+
 body {
   margin: 0;
+  font-family: $body-font;
+  font-size: $font-size;
+}
+
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  width: 100%;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  cursor: pointer;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding: 16px 16px 40px;
+  }
+
+  &__title {
+    font-size: $font-size + 2px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-main-accent;
+    padding-bottom: 4px;
+  }
+
+  &__code {
+    width: 100%;
+    color: $color-secondary;
+    padding-bottom: 16px;
+    line-height: 14px;
+  }
+
+  &__stars {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 23px;
+
+    &-count {
+      text-align: right;
+      font-style: normal;
+      font-weight: 400;
+      color: $color-main-accent;
+    }
+  }
+
+  &__price {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 16px;
+
+    &-tag {
+      font-size: $font-size + 2px;
+      color: $color-secondary;
+    }
+
+    &-value {
+      font-size: $font-size + 6px;
+      font-weight: 700;
+      color: $color-main-accent;
+      display: flex;
+      vertical-align: bottom;
+    }
+  }
+
+  .card__buy-button {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 5px;
+    background-color: $color-blue-accent;
+    color: #fff;
+    font-size: $font-size + 4px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+    line-height: normal;
+    padding: 13px 0;
+
+    &:hover {
+      color: $color-blue-accent;
+      background-color: #fff;
+      border: 1px solid $color-blue-accent;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.stars__star:last-of-type {
+  margin-right: 0;
+}
```

Comment:
you can nest those inside .stars, according to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) point 11 there should be 1 file for each BEM block - so 1 for stars and 1 for card